src/lib.rs

```diff
@@ -257,6 +258,32 @@ impl ParserConfig {
         self
     }
 
+    /// Sets whether multiple spaces are allowed as delimiters in response status lines.
+    ///
+    /// Note that if this option is enabled, the extra spaces for the second delimiter will be
+    /// included in the parsed reason phrase.
```

Comment:
This feels a bit unclean, but prevents adding a loop after parsing the status code. It would of course be straightforward to add a `skip_spaces()` call before checking for the reason phrase or line ending if we're comfortable with the performance implications.